dbt/models/iasworld/schema.yml

```diff
@@ -26,6 +31,10 @@ sources:
       - name: htagnt
       - name: htdates
       - name: htpar
+        freshness:
+          filter: taxyr = date_format(current_date, '%Y')
```

Comment:
**suggestion (blocking):** I _think_ we actually want all of these to be `taxyr = date_format(current_date, '%Y') - 1`. Since prop taxes are paid in arrears, very often the "current" or "working" year is actually the prior year. Note that:

- sqoop is setup to fetch `current_year - 2` every day
- Rolling over to a new calendar year doesn't necessarily mean rolling over to a new tax year in the system. So it's likely that these tests would start failing on Jan 1, 2024 since there are no records in the filter. This is fixed by using `current_year - 1`.

@wrridgeway Any thoughts here?

Reply:
Makes sense to me! I'll go ahead and bump it back to `current_year - 1` while we discuss, it should be very easy to tweak if we end up deciding on a different threshold.

Reply:
Yes, current year, as much as i'd love to use it, would bunk things up come January. When the new year's roll is generated currently does not seem to be automated but rather a manual process. So my vote is `current_year -1` as well.